### PR TITLE
Update the Blender collision empties tip for Blender 2.80+

### PR DESCRIPTION
**Pull request by mshayden1998**
My purpose is to update the steps guide on "Note" (from line 432 and forward) to matches the current Blender's version plus an observation for users who still on Blender's old versions.

Obs.: this is my first pull request. Did I made it right?

### CODE REVIEW

File: getting_started/workflow/assets/importing_scenes.rst [hunk 1]
```diff
@@ -431,8 +431,11 @@ reliability.
 
 .. note::
 
-    For better visibility in Blender's editor, you can set the "X-Ray" option
-    on collision empties and set some distinct color for them in Blender's
+    For better visibility on Blender's editor, you can set the "X-Ray" option
+    on collision empties and set some distinct color for them by going on
+    **Edit > Preferences > Themes > 3D Viewport > Empty**.
+    
+    Obs.: if you are using a Blender's version below 2.8, maybe the steps are
```
Comment (Calinou) [inline getting_started/workflow/assets/importing_scenes.rst:438]:
```suggestion
    If using Blender 2.79 or older, change the following setting instead:
```

**Review by Calinou — CHANGES_REQUESTED** [review]

File: getting_started/workflow/assets/importing_scenes.rst [hunk 2]
```diff
@@ -431,8 +431,11 @@ reliability.
 
 .. note::
 
-    For better visibility in Blender's editor, you can set the "X-Ray" option
-    on collision empties and set some distinct color for them in Blender's
+    For better visibility on Blender's editor, you can set the "X-Ray" option
+    on collision empties and set some distinct color for them by going on
```
Comment (pycbouh) [inline getting_started/workflow/assets/importing_scenes.rst:435]: "going to", I think?

**Review by pycbouh — CHANGES_REQUESTED** [review]

File: getting_started/workflow/assets/importing_scenes.rst (hunk 2 above)
Comment (mshayden1998) [reply getting_started/workflow/assets/importing_scenes.rst:435]: > "going to", I think?

Yeah, sounds better. . _.

Comment (Calinou) [inline getting_started/workflow/assets/importing_scenes.rst:435]:
```suggestion
    on collision empties and set some distinct color for them by changing```

File: getting_started/workflow/assets/importing_scenes.rst [hunk 3]
```diff
@@ -431,8 +431,11 @@ reliability.
 
 .. note::
 
-    For better visibility in Blender's editor, you can set the "X-Ray" option
-    on collision empties and set some distinct color for them in Blender's
+    For better visibility on Blender's editor, you can set the "X-Ray" option
+    on collision empties and set some distinct color for them by changing```
```
Comment (Calinou) [inline getting_started/workflow/assets/importing_scenes.rst:435]:
```suggestion
    on collision empties and set some distinct color for them by changing
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**